io/net/bonding.py

```diff
@@ -520,6 +526,19 @@ def test_cleanup(self):
             except Exception:
                 self.log.info("backup file not availbale, could not restore file.")
 
+        detected_distro = distro.detect()
+        if detected_distro.name == "rhel":
+            cmd = "systemctl restart NetworkManager.service"
+        elif detected_distro.name == "Ubuntu":
+            cmd = "systemctl restart networking"
+        else:
+            cmd = "systemctl restart network"
```

Comment:
@TasmiyaNalatwad are we not using common lib service method for services to restart?

Reply:
@Naresh-ibm  Yes we can use the util function, but the function can be used to restart the service only at host machine, and to achieve restart of service at peer machine i need to write these lines again. To avoid the duplicating the code lines wrote only one block of code to achieve the restart of service at both the lpars.

Reply:
I have not seen a case where peer service is restarted,  that would disturb rhe peer set up of